thunder/executors/nvfuserex_impl.py

```diff
@@ -2194,3 +2194,30 @@ def map_inside_replacement(x: Any) -> None:
     elapsed_time_millis = elapsed_time_ns // 1000000
     rrctrace.set_provenance(TraceProvenance(f"Remove redundant casts (took {elapsed_time_millis} milliseconds)"))
     return rrctrace
+
+
+def _matmul_check(
+    a: TensorProxy,
+    b: TensorProxy,
+) -> bool:
+    if nv_version < LooseVersion("0.2.2"):
+        return False
```

Comment:
Since this feature is guarded behind the `nv_enable_matmul` option. Let's raise an error for not new enough nvFuser installed.

Reply:
Since we test using nvFuser 0.2.0 as well on the CI, I am letting the check reject the matmul instead.

Reply:
Sure we have 0.2.0 in the CI and we can skip tests based on the nvFuser version used. Warning from the compile option is good enough.